entrypoint.sh

```diff
@@ -100,37 +99,131 @@ function super_lint() {
   /action/lib/linter.sh
 }
 
+function get_files_matching_filetype() {
```

Comment:
This function compiles a list of files to be linted based on filetype metadata passed in from the linters.txt file.